instance/models/instance.py

```diff
@@ -157,6 +157,11 @@ class Instance(ValidateModelMixin, models.Model):
         blank=True,
         help_text='Custom tags associated with the instance.',
     )
+    accepted_privacy_policy = models.DateTimeField(
```

Comment:
@jamestait, just thinking out loud, while we could block the spawning of new appservers unless the privacy policy is accepted, I am not sure if that is the best way forward because the customers almost never spawn new appservers after the initial trials to get their theming right.

Reply:
The approach that appears to have been decided upon is to ask existing customers to agree to the Privacy Policy via email.